lib/apollo-federation/has_directives.rb

```diff
@@ -9,8 +9,8 @@ def add_directive(name:, arguments: nil)
       @federation_directives << { name: name, arguments: arguments }
     end
 
-    def to_graphql
-      field_defn = super # Returns a GraphQL::Field
+    def to_graphql(**args)
```

Comment:
Thanks for linking to [this](https://github.com/rmosolgo/graphql-ruby/commit/f449669e54601a1ad213c97767c810df83a92414). I know the tests were failing but that was in the 1.9.x test suite We [removed support](https://github.com/Gusto/apollo-federation-ruby/pull/162/files) for 1.9.x and now tests can pass without adding args. 

But the key question IMO is what are the args for? From reading the commit It seems like args were actually added to support passing `silence_deprecation_warnings: true`  to `to_graphql`. 

1. Should we be using `to_graphql(**args, **kwargs)`? This captures both positional and keyword args to the function?
2. With our library patching `to_graphql`, does the silencing of args still work as expected?